taskfiles/utils.yml

```diff
@@ -103,3 +103,47 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  # ===
+  # C++ LINTING UTILS
+  # ===
+
+  # @param {string} FLAGS - The flags to pass to clang-format
+  # @param {string} LINT_VENV_DIR - Directory with venv fo linting
+  # @param {[]string} SRC_DIRS - The directories to start linting on.
+  # clang-format.
+  clang-format:
+    internal: true
+    ignore_error: true
```

Comment:
We don't want to ignore errors, right? Ditto for clang-tidy.

Reply:
If we don't ignore errors, then if this is trying to run on multiple paths. If the first path has a tidy error or format error, then it won't finish running on any of the other paths.

Reply:
I see. Ignoring errors won't work for our automated workflows though since the task won't fail. Instead of using a task-based for-loop, can we use a Go template `range` action to create a concatenation of the map's entries?